cg/meta/workflow/microsalt/microsalt.py

```diff
@@ -89,13 +89,16 @@ def get_config_path(self, filename: str) -> Path:
 
     def get_job_ids_path(self, case_id: str) -> Path:
         project_id: str = self.get_project_id(case_id)
```

Comment:
```suggestion
        """ Get job id path and unlink if path exists. """
        project_id: str = self.get_project_id(case_id)
```

Reply:
Extracted a helper method making the code self documenting